api/services/ElasticSearchService.js

```diff
@@ -43,18 +43,44 @@ module.exports = {
             }
           ]
         }
-      }
+      },
+      lastMonthDownloads: function(n){
+        return {
+        "size":"10000",
+        "fields":["datetime","ip_id","package","version"],
+        "sort":[ {"ip_id":{"order":"asc","ignore_unmapped" : true}},
+                    {"datetime":{"order":"asc","ignore_unmapped" : true}}],
+        "query":{
+            "bool": {
+                  "filter": [
+                    {
+                        "term": { "_type": "stats" }
+                    },
+                    {
+                      "range": {
+                          "datetime":  {
+                              "gte" : "now-"+(n+1)+"d/d",
+                              "lt" :  "now-"+n+"d/d"
+                          }
+                      }
+                    }
+                  ]
+                  }
+              }
+
+          }
+        }
     }
   },
-
+      
   lastMonthPercentiles: function() {
     var body = {
       "query": ElasticSearchService.queries.filters.lastMonthStats,
       "size": 0, // do not retrieve data, we are only interested in aggregation data
       "aggs" : _.pick(ElasticSearchService.queries.aggregations, ['download_per_package', 'download_percentiles'])
     };
 
-    return es.search({
+    es.search({
```

Comment:
- [ ] Is deleting this `return` intentional ?
it will likely break the functions that rely on this one.